N.B.
```
Traceback (most recent call last):
  File "/home/runner/work/sympy/sympy/sympy/core/tests/test_args.py", line 74, in test_all_classes_are_tested
    assert not failed, "Missing classes: %s.  Please add tests for these to sympy/core/tests/test_args.py." % ", ".join(failed)
AssertionError: Missing classes: sympy.stats.drv_types.FlorySchulzDistribution.  Please add tests for these to sympy/core/tests/test_args.py
```

--- a/sympy/core/tests/test_args.py
+++ b/sympy/core/tests/test_args.py
@@ -1595,7 +1595,7 @@ def test_sympy__stats__drv_types__NegativeBinomialDistribution():
     from sympy.stats.drv_types import NegativeBinomialDistribution
     assert _test_args(NegativeBinomialDistribution(.5, .5))
 
-def test_sympy_stats_drv_types_FlorySchulzDistribution():
+def test_sympy__stats__drv_types__FlorySchulzDistribution():
     from sympy.stats.drv_types import FlorySchulzDistribution
     assert _test_args(FlorySchulzDistribution(.5))
 

--- a/sympy/core/tests/test_args.py
+++ b/sympy/core/tests/test_args.py
@@ -1595,6 +1595,9 @@ def test_sympy__stats__drv_types__NegativeBinomialDistribution():
     from sympy.stats.drv_types import NegativeBinomialDistribution
     assert _test_args(NegativeBinomialDistribution(.5, .5))
 
+def test_sympy_stats_drv_types_FlorySchulzDistribution():
+    from sympy.stats.drv_types import FlorySchulzDistribution
+    assert _test_args(FlorySchulzDistribution(.5))
 
 def test_sympy__stats__drv_types__PoissonDistribution():
     from sympy.stats.drv_types import PoissonDistribution

--- a/sympy/stats/drv_types.py
+++ b/sympy/stats/drv_types.py
@@ -163,7 +163,7 @@ def FlorySchulz(name, p):
     >>> X = FlorySchulz("x", p)
 
     >>> density(X)(z)
-    (4/5)**(z - 1)*z/25)
+    (4/5)**(z - 1)*z/25
 
     >>> E(X)
     9

--- a/sympy/stats/tests/test_discrete_rv.py
+++ b/sympy/stats/tests/test_discrete_rv.py
@@ -40,6 +40,7 @@ def test_Poisson():
         assert isinstance(E(2*x, evaluate=False), Expectation)
     # issue 8248
     assert x.pspace.compute_expectation(1) == 1
+
 def test_FlorySchulz():
     p = S.One / 5
     x = FlorySchulz('x' , p)

--- a/sympy/stats/tests/test_discrete_rv.py
+++ b/sympy/stats/tests/test_discrete_rv.py
@@ -9,7 +9,7 @@ from sympy.stats import (P, E, variance, density, characteristic_function,
                          where, moment_generating_function, skewness, cdf,
                          kurtosis, coskewness)
 from sympy.stats.drv_types import (PoissonDistribution, GeometricDistribution,
-                                   Poisson, Geometric, Hermite, Logarithmic,
+                                   FlorySchulz, Poisson, Geometric, Hermite, Logarithmic,
                                     NegativeBinomial, Skellam, YuleSimon, Zeta,
                                     DiscreteRV)
 from sympy.stats.rv import sample
@@ -40,6 +40,12 @@ def test_Poisson():
         assert isinstance(E(2*x, evaluate=False), Expectation)
     # issue 8248
     assert x.pspace.compute_expectation(1) == 1
+def test_FlorySchulz():
+    p = S.One / 5
+    x = FlorySchulz('x' , p)
+    assert E(x) == 9
+    assert variance(x) == 40
+    assert density(x)(2) - 8/125 < 0.001
 
 @slow
 def test_GeometricDistribution():

